zerver/webhooks/zendesk/doc.md

```diff
@@ -1,51 +1,50 @@
+# Zulip Zendesk integration
+
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. Construct a webhook URL like the following:
-
-    `{{ api_url }}/v1/external/zendesk?ticket_title={% raw %}{{ ticket.title }}&ticket_id={{ ticket.id }}{% endraw %}`
+1. {!generate-webhook-url-basic.md!}
 
 1. {!append-channel-name.md!}
 
-1. Next, in Zendesk, open your **Admin** view via gear in the bottom-left
-    corner. In the **Admin** view, click on **Extensions**, then click
-    **add target**.
+1. In Zendesk, click the **gear** (<i class="fa fa-cog"></i>) icon in the
+    bottom-left corner, click on **Extensions**, and then click **add target**.
 
     ![](/static/images/integrations/zendesk/001.png)
     ![](/static/images/integrations/zendesk/002.png)
 
-1. From there, click **URL target**. Fill in the form like this:
```

Comment:
Removed connecting words like these from most steps, as per the [guideline in our RTD pages](https://zulip.readthedocs.io/en/latest/documentation/integrations.html#:~:text=Cut%20unnecessary%20words.%20For%20example%2C%20do%20not%20start%20steps%20with%20transition%20words%20like%20%E2%80%9CNext%E2%80%9D%2C%20%E2%80%9CFirst%E2%80%9D%2C%20%E2%80%9CNow%E2%80%9D%2C%20etc.%20Each%20step%20should%20be%20structurally%20independent.).